readthedocs/urls.py

```diff
@@ -44,6 +44,7 @@
     url(r'^accounts/', include('readthedocs.profiles.urls.private')),
     url(r'^accounts/', include('allauth.urls')),
     url(r'^notifications/', include('readthedocs.notifications.urls')),
+    url(r'^accounts/gold/', include('readthedocs.gold.urls')),
```

Comment:
Other question is do we move Gold out as well, and keep it gated in a similar way?

Reply:
Eventually, +1 from me. We can address this separately. Feel free to open a ticket to track the work.